config/samples/resources/alloydbcluster/regular-cluster/kms_v1beta1_kmskeyring.yaml

```diff
@@ -15,6 +15,6 @@
 apiVersion: kms.cnrm.cloud.google.com/v1beta1
 kind: KMSKeyRing
 metadata:
-  name: alloydbcluster-dep
+  name: alloydbcluster-dep-regular
 spec:
-  location: us-central1
\ No newline at end of file
+  location: us-east1
```

Comment:
Out of curiosity, why do we want to change the location? Does us-east1 offer better capacity for AlloyDB compared to us-central1?

Reply:
This is mostly to ensure that while running tests on samples in parallel, we do not run out of quota. As there is a quota on number of clusters per region per project